src/android/app/build.gradle

```
@@ -96,7 +96,7 @@ android {
 
     externalNativeBuild {
         cmake {
-            version "3.18.1"
+            version "3.25.1"
```

```suggestion
            version "3.22.1"
```
Preferably stick to versions available in the Android SDK, so gradle can automatically fetch them when they are not available